externals/iframeResizer.contentWindow.min.js

```diff
@@ -0,0 +1,10 @@
+/*! iFrame Resizer (iframeSizer.contentWindow.min.js) - v4.3.1 - 2021-01-11
+ *  Desc: Include this file in any page being loaded into an iframe
+ *        to force the iframe to resize to the content size.
+ *  Requires: iframeResizer.min.js on host page.
+ *  Copyright: (c) 2021 David J. Bradshaw - dave@bradshaw.net
+ *  License: MIT
```

Comment:
>  License: MIT [](http://example.com/codeflow?start=2&length=14)

This license is good for us to use - right?

@southpolesteve  - do we have to declare all the open soure packages we use anywhere? Or because, we ourselves are open source, we dont have to bother?

Reply:
I think it should be okay. We're not modifying this library and are using it as is.